libminifi/test/unit/ExpectedTest.cpp

```diff
@@ -558,6 +558,15 @@ TEST_CASE("expected orThrow") {
   nonstd::expected<int, std::string> unexpected{nonstd::unexpect, "hello"};
   nonstd::expected<int, std::string> expected{5};
 
+
   REQUIRE_THROWS_WITH(std::move(unexpected) | utils::orThrow("should throw"), "should throw, but got hello");
   CHECK((expected | utils::orThrow("should be 5")) == 5);
 }
+
+// This fails to compile with std::expected on GCC 15.1 due to https://gcc.gnu.org/bugzilla/show_bug.cgi?id=119714
+TEST_CASE("") {
```

Comment:
Let's give this test a name; for example, the comment would work:
```suggestion
TEST_CASE("This fails to compile with std::expected on GCC 15.1 due to https://gcc.gnu.org/bugzilla/show_bug.cgi?id=119714") {
```